.gitignore

```diff
@@ -2,6 +2,7 @@
 .cmake-format.yaml
 nmodlconf.h.in
 nrnconf.h.in
+build
```

Comment:
I would not ignore this. It is present only with a specific build style (and multiple builds get different names).
Not relevant here, but nmodlconf.h.in and nrnconf.h.in are no longer built (by cmake) in the nrn folder.